compiler/rustc_middle/src/ty/codec.rs

```diff
@@ -483,13 +475,9 @@ macro_rules! implement_ty_decoder {
                 }
 
                 #[inline]
-                fn read_raw_bytes_into(&mut self, bytes: &mut [u8]) -> Result<(), Self::Error> {
+                fn read_raw_bytes_into(&mut self, bytes: &mut [u8]) -> () {
```

Comment:
Can you remove the `-> ()`?

Reply:
Done, here and in one other place.

Reply:
The explicit return `()` https://github.com/rust-lang/rust/pull/93066/files#diff-dd7fe6a03ae21dfa7be21cfbf2dae90ee076550976b89acff955394306ea500fR2299 can also be removed, here as well as a couple other places which also used to return `Ok(())`.